src/Git/GetVersionCollectionFromGitRepository.php

```diff
@@ -30,7 +30,7 @@ public function fromRepository(CheckedOutRepository $checkedOutRepository) : Ver
             ->mustRun()
             ->getOutput();
 
-        return VersionsCollection::fromArray(array_filter(
+        return new VersionsCollection(...array_filter(
```

Comment:
An `array_values()` is needed here. To test this, try an array of `[1.0.0, 1.potato, 2.0.0]` and you will get a TypeError if I remember correctly (test is to be added)

Reply:
Seems OK to me: https://3v4l.org/d5RvE

We test invalid at `testFromRepositoryIgnoresInvalidVersions`

Reply:
Hmm, interesting. I guess it crashes only when there are non-numeric keys then :+1: